### ISSUE TITLE
About training_cache and training time

### ISSUE THREAD

**Issue by Whale-ice** [autonomousvision/navsim#39]
Hello, I would like to ask if a large amount of time is required to create the training_cache when using navtrain for training? When I use navtrain, it usually takes around 20 hours to generate the training_cache. Are there any faster methods available? Also, I would like to know how long it took you to train Transfuser and UniAD. Thank you!

**Comment by DanielDauner** [comment]
Hi @Whale-ice,

Caching the training set can take quite some time, depending on how computationally demanding your pre-processing is per sample. For example, the TransFuser baseline needs to pre-process the model input and all targets for the auxiliary tasks (e.g., the semantic maps, and bounding boxes). We still recommend caching the training set due to added stability and speed during training. The best practice is to parallelize the caching with the `worker=ray_distributed` argument.

TransFuser took about 24 hours on a single GPU (i.e. 100 epochs of training). UniAD took longer because a pre-training stage was involved. @sephyli might have further information. I am closing this issue for now.

All the best!
Daniel

**Comment by Whale-ice** [comment]
Thank you for your reply! 
Now, I have another question: i eval the ego_mlp_agent on nevtest with ckpt from https://huggingface.co/autonomousvision/navsim_baselines/tree/main/ego_status_mlp/seed 0  but get pdms 3.4*e-5
<img width="1220" alt="image" src="https://github.com/user-attachments/assets/1f68df11-a4d5-4a45-b19b-74163144380f">
which it different from navsim benchmark

I set the train_test_split: navtest  and total 12k scenario.

**Comment by Whale-ice** [comment]
oh, i find the same question as issue 28 and i settle it !
